source/clear-linux/get-started/virtual-machine-install/vmware-player-2.rst

```diff
@@ -0,0 +1,469 @@
+.. _vmware-player:
```

Comment:
This label is already used in vmware-player.rst, have the label match the title so we can easily reference this doc. Also, I suggest we change the title to something that differentiates this doc from vmware-player.rst.